boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280_cpuppr.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf9280pdk/nrf9280/cpuppr
+name: nRF9280-DK-nRF9280-PPR
+type: mcu
+arch: riscv
+toolchain:
+  - zephyr
+sysbuild: true
+ram: 16
+flash: 64
```

Comment:
I believe `flash` should equal `ram` in the non-XIP case.

Reply:
Updated